README.md

```diff
@@ -1,2 +1,77 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
 # Metrics-Web-App
 Building a mobile web application to check a list of metrics (numeric values) that will be created by making use of React and Redux.
+
+## Built With
+- React
+- Redux
+- JavaScript
+- CSS
+- Bootstrap
+- Redux Toolkit
+- ACNH API (v1)
+
+## Live Demo
+[Vintage](https://vintage-art-collection.netlify.app/)
+
+## Loom Video Link
+[Project Documentation](https://www.loom.com/share/8d32484262494a13b3ddcfea516ebf6e)
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- Have a Desktop or Laptop Computer running on with Windows, Mac OS or Linux operating system.
+- Have git installed on the Computer
+- Have a browser install on the Computer
+
+### Setup
+
+- If it suits you, on your local computer, create a directory in your preferred location (e.g in the Desktop or Document directory)
+
+### Install
+- To clone the project on your machine: Type `git clone https://github.com/NahnahAJ/Metrics-Web-App.git` if your're using https with git or `git@github.com:NahnahAJ/Metrics-Web-App.git` if you prefer ssh and hit enter.
+- To install project dependencies _(a packet manager is required)_:
+`npm install`
+- To build project files in development mode _(the destination folder is /dist)_:
+`npm run build`
+- To run the project on localhost:3000
+`npm run start`
+ 
+
+### Usage
+
+- You can view the document in your preferred or default browser.
+
+
+## Author
+
+👤 **Nahnah Ahjuwah**
+
+- GitHub: [@NahnahAJ](https://github.com/NahnahAJ)
+- Twitter: [@NahnahAhjuwah](https://https://twitter.com/NahnahAhjuwah)
+- LinkedIn: [Felicia-Awuah](https://www.linkedin.com/in/felicia-awuah-gyedua/)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+- Microverse
+- Original design idea by [Nelson Sakwa on Behance](https://www.behance.net/sakwadesignstudio)
+
+# Attribution
+[Creatve Commons Lincense of the Design](https://creativecommons.org/licenses/by-nc/4.0/)
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Great jon working on the README but since Microverse updated the template so kindly update it to be the same 🔥🔥💯💯💯 you can see it [here](https://github.com/microverseinc/readme-template) ✨✨🙏🏻